docs/terminal/amberleyoakey.md

```diff
@@ -4,7 +4,127 @@
+
+### OK TCU
+#### Restricted Areas
+By Default YBOK owns all of the R625 Restricted Areas, detailed below:
+
+- R654A (`SFC`-`A065`)  
+- R6545B (`SFC`-`A065`)  
+- R654C (`A065`-`F125`)  
+- R654D (`A065`-`F125`)  
+
+#### Diagram
+
+<figure markdown>
+![OK TCU Airspace Diagram](img/oktcu.png){ width="1060" }
+  <figcaption>OK TCU Airspace Diagram</figcaption>
+</figure>
+
+### OK ADC
+OK ADC owns the CLass C airspace within 5 DME of the OK VOR from `SFC` to `A025`. This airspace is the circuit area.
+
+
+## YAMB Aerodrome
+### Coded Departures (Fast Jets)
+Visual Departures are commonly requested which are often in the form of a coded departure (e.g. BYRON7 departure), which can be found in the AD2 Sups Page for Amberley. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane avoiding civilian traffic inbound to Brisbane. 
+
+
+Aircraft will make a visual departure in the circuit direction and fly overhead YAMB to begin tracking for their initial waypoint.
+
+!!! example
+    WOLF03 was assiged the BYRON7 departure with their intial clearance.  
+    **AM ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
+    **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  
+
+| Departure | Initial Constraint | Route |
+| --------- | ----------| --------- |
+| Northern 1 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
+| Central 1 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
+| Byron 7 | `F190` at BINUP | COWIE KIWEE ZANET LOTMA SEMAJ |
+
+!!! note
+    Charts for the Coded Departures may be in YAMB AD2 Supplements 6.2.25.11 available here: [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+
+## Charts
 !!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+
+## YAMB Coordination
+### AMB ADC
+
+'Next' coordination is required from AM ADC to AM TCU for all aircraft.
+
+!!! example
+    <span class="hotline">**AM ADC** -> **AM TCU**</span>: "Next, ASY01, runway 33"  
+    <span class="hotline">**AM TCU** -> **AM ADC**</span>: "ASY01, Assigned Heading Right 030, unrestricted"  
+    <span class="hotline">**AM ADC** -> **AM TCU**</span>: "Assigned Heading Heading Right 030, ASY01"  
+
+### Enroute
+#### Departures
+Departing aircraft require heads up coordiantion to INL(All) as soon as practical (aircraft becomes airborne in most cases). Note since there are no assignable level assign the most pracitcal (ie. `F180` due airspace celling or coordinate higher for high performance jets).
```

Comment:
"All aircraft from AM/OK TCU to INL(All) and BN TCU require Heads-up coordination prior to the boundary, however, as soon as practical (when the aircraft becomes airborne) is preferred."

Remove all else